EAD/EADSession4/WebContent/WEB-INF/web.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:web="http://xmlns.jcp.org/xml/ns/javaee"
+	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd">
+	<context-param>
+		<description>Location of Stored Images</description>
+		<param-name>profileImage</param-name>
+		<param-value>
+			C:\Users\Mohit\eclipse-workspace\GET2018\src\GET2018\EAD\EADSession4\Images
```

Comment:
Hardcoded entries can be removed from this file. (html files can be added in welcome-file tags) and images can be added in images folder with other images